ansible/roles/operator-pipeline/templates/openshift/tasks/read-config.yaml

```diff
@@ -32,16 +34,30 @@ spec:
 
         if [ ! -f "$CONFIG_PATH" ]; then
             echo "Config file $CONFIG_PATH does not exist."
-            echo -n "replaces" | tee $(results.upgrade-graph-mode.path)
+            echo "replaces" | tee $(results.upgrade-graph-mode.path)
+            echo "false" | tee $(results.fbc-enabled.path)
             exit 0
         fi
 
         cat $CONFIG_PATH
 
-        # Parse the upgrade graph mode with default value "replaces"
-        UPGRADE_GRAPH_MODE=`cat $CONFIG_PATH | yq -r '.updateGraph // "replaces"'`
-
-        # Remove the "-mode" suffix
-        UPGRADE_GRAPH_MODE=`echo $UPGRADE_GRAPH_MODE | sed 's/-mode$//'`
+        # Parse the fbc enablement status
+        FBC_ENABLEMENT=`cat $CONFIG_PATH | yq -r '.fbc.Enabled // "false"'`
+
+        if [ "$FBC_ENABLEMENT" == "true" ]; then
+            if [ "$(cat $CONFIG_PATH | yq -r '.updateGraph')" != "null" ]; then
+                echo "FBC is enabled, updateGraph cannot be defined."
+                exit 1
```

Comment:
The AC doesn't say that upgradeGraph is forbidden: it says it's not required.

I have no strong preference for either one: just pointing out the mismatch.

Reply:
This decision is based on offline discussion with @Allda I think this is more clear, not to let users create accidental mismatch providing both options. But also have no strong opinion and can simplify this just to read the config values if anyone have some reasoning to dismiss current implementation.